src/types.rs

```diff
@@ -171,12 +171,26 @@ impl ToGeo<f64> for TrackSegment {
     }
 }
 
+// A Version of geo::Point that has the Default trait implemented
+#[derive(Clone, Debug)]
+struct OurPoint {
```

Comment:
Would using a [one-field tuple](https://doc.rust-lang.org/1.6.0/book/structs.html#tuple-structs) here be preferable/cleaner? Also, please rename it to `GpxPoint` and add a comment explaining why we added default in the first place.

```rust
struct GpxPoint(Point);

impl Default for GpxPoint {
    fn default() -> GpxPoint {
        GpxPoint(Point::new(0., 0.))
    }
}
```